.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -41,23 +89,49 @@ If you need to submit a pr for any of these repos during this period, you should
+> https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes
 - [ ] Generate release pipeline jobs for the new build scripts repo release tag, helper repo release tag and aqaReference: https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/release-build-pipeline-generator/build?delay=0sec
+
+**Wait For All Of The Above To Complete Successfully Before Proceeding!**
+
+- [ ] Log a helpdesk ticket with EF , to get all test materials updated
 - [ ] TC: Run the ProcessCheckMultiNode process cleaning job on all ci.role.test nodes, to ensure healthy state, verify all nodes successful: https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/build?delay=0sec
 - [ ] TC: Run the Setup_JCK_Run_Multinode job with CLEAN_DIR=true (to purge any old release contents/results) on all ci.role.test nodes, this will extract the jck_run folder with all the temurin.jtx exclude files, verify all nodes successful : https://ci.eclipse.org/temurin-compliance/job/Setup_JCK_Run_Multinode/build?delay=0sec
+- [ ] Check the nagios server to ensure there are no critical infrastructure issues
 - [ ] **Trigger a trial release pipeline dry-run** to ensure less surprises on release day (typically against a milestone build), see here for [details](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#auto-way---before-release-week-auto-test)
 
 - [ ] Confirm successful trial release pipelines and successful jck completion
-- [ ] Calculate the "expected" openjdk build tags for the releases being published, and update all the JDKnn_BRANCH values in the testenv.properties file for the aqa-tests release branch, eg: https://github.com/adoptium/aqa-tests/blob/v0.9.6-release/testenv/testenv.properties 
+- [ ] Calculate the "expected" openjdk build tags for the releases being published, and update all the JDKnn_BRANCH values in the testenv.properties file for the aqa-tests release branch, eg: https://github.com/adoptium/aqa-tests/blob/v0.9.6-release/testenv/testenv.properties
 
 -------
 
 Release Week Checklist:
 
 - [ ] **Add website banner** (_automate_* via github workflow in website repository) - Announce that we target releases to be available within 48-72 hours of the GA tags being available
+- [ ]   -- Check All Nodes Online ( Needs adding to release checklist ) - https://ci.eclipse.org/temurin-compliance/label/ci.role.test/
+- [ ]  Run https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/ -- with defaults
+- [ ] Run Setup_JCK_Multinode with CLEAN_DIR=true for ( ci.role.test )
+- [ ] Disable Setup JCK_MUltinode To Ensure Test Evidence Is Not Lost
+- [ ] Check the nagios server to ensure there are no critical infrastructure issues
+- [ ] Create The Github Issues for tracking progress against each Java version
+
+The following steps have all been automated ( with the exception of manually triggering the JDK8 AARCH32 build, which must be done seperately when the tags are available )
```

Comment:
Think this needs some structural adjustment. Naively it reads everything following this statement is automated all the way to declaration of the release as complete.

Reply:
Adjusted appropriately, to reduce confusion.